fix: allow full range of ASCII and EBCDIC characters accepted by Nacha

adamdecaf opened the pull request:
The 2022 Nacha rules state that Valid Characters are:

> 0-9, A-Z, a-z, space, EBCDIC values greater than hexadecimal "3F",
> ASCII values greater than hexadecimal "1F".

- [ASCII Characters](https://en.wikipedia.org/wiki/ASCII#Character_set)
- [EBCDIC Characters](https://en.wikipedia.org/wiki/EBCDIC)

adamdecaf committed:
test: check which bar characters are allowed

alovak commented on validators_test.go:
```diff
@@ -175,6 +175,22 @@ func TestValidators__isAlphanumeric(t *testing.T) {
+func TestValidators__isAlphanumericExamples(t *testing.T) {
+	v := validator{}
+
+	validCases := []string{`|`}
```
I'm curious, is `|` alphanumeric (letters and numbers)? I guess it is in ACH. Right?

adamdecaf replied:
`|` is `0x7C` [in ASCII](https://en.wikipedia.org/wiki/ASCII#Character_set) which is allowed by Nacha.

adamdecaf committed:
fix: allow full range of ASCII and EBCDIC characters accepted by Nacha rules

The 2022 Nacha rules state that Valid Characters are:

> 0-9, A-Z, a-z, space, EBCDIC values greater than hexadecimal "3F",
> ASCII values greater than hexadecimal "1F".
